### PR TITLE
Update aa french translations

### PR DESCRIPTION
**Pull request by Jean-BaptisteC**

### CODE REVIEW

**Comment by AndrewShkrob** [comment]
Android Lint error can be skipped. It is fixed in Permissions Screen PR

File: data/strings/strings.txt [hunk 1]
```diff
@@ -27019,28 +27019,33 @@
...
     comment = Android Auto connected
     tags = android
     en = You are connected to Android Auto
+    fr = Vous êtes connecté à Android Auto
     ru = Вы подключены к Android Auto
 
   [aa_continue_on_phone]
     comment = Button to show maps on the phone screen instead of a car
     tags = android
     en = Continue on phone
```
Comment (biodranik) [inline data/strings/strings.txt:27042]:
```suggestion
    en = Continue on the phone
```

File: data/strings/strings.txt [hunk 2]
```diff
@@ -27019,28 +27019,33 @@
...
+    fr = Continuer sur le téléphone
     ru = Продолжить в телефоне
 
   [aa_continue_in_car]
     comment = Button to show maps on the car screen instead of a phone. Must be no more than 18 symbols!
     tags = android
     en = Continue in car
+    fr = Continuer dans la voiture
```
Comment (biodranik) [inline data/strings/strings.txt:27050]: 26 chars

File: data/strings/strings.txt [hunk 3]
```diff
@@ -27019,28 +27019,33 @@
...
     comment = Button to show maps on the phone screen instead of a car
     tags = android
     en = Continue on phone
+    fr = Continuer sur le téléphone
     ru = Продолжить в телефоне
 
   [aa_continue_in_car]
     comment = Button to show maps on the car screen instead of a phone. Must be no more than 18 symbols!
     tags = android
     en = Continue in car
```
Comment (biodranik) [inline data/strings/strings.txt:27049]:
```suggestion
    en = To the car screen
```

File: data/strings/strings.txt [hunk 4]
```diff
@@ -27019,28 +27019,33 @@
...
 
   [aa_continue_on_phone]
     comment = Button to show maps on the phone screen instead of a car
     tags = android
     en = Continue on phone
+    fr = Continuer sur le téléphone
     ru = Продолжить в телефоне
 
   [aa_continue_in_car]
     comment = Button to show maps on the car screen instead of a phone. Must be no more than 18 symbols!
```
Comment (biodranik) [inline data/strings/strings.txt:27047]: Is it AA limit?

Comment (AndrewShkrob) [reply data/strings/strings.txt:27047]: Yes :(

File: data/strings/strings.txt (hunk 2 above)
Comment (Jean-BaptisteC) [reply data/strings/strings.txt:27050]:
```suggestion
    fr = Dans la voiture
```

File: data/strings/strings.txt (hunk 4 above)
Comment (Jean-BaptisteC) [reply data/strings/strings.txt:27047]: I suggest `fr = Aller sur le téléphone`

Comment (biodranik) [reply data/strings/strings.txt:27047]: @AndrewShkrob can this check be added to our strings linter?

Comment (AndrewShkrob) [reply data/strings/strings.txt:27047]: It appears that I might have confused some details. This button is meant to be utilized on the device's screen, hence there should be no limit. The restriction regarding the button limit should pertain to the button within AA instead. I will take another look to ensure clarity.

**Comment by Jean-BaptisteC** [comment]
PTAL

**Comment by biodranik** [comment]
1. Andrew need to check the limit.
2. It would be great to add this check to our strings regeneration script or as a separate checker tool.
3. Android Lint check fails.

**Review by rtsisyk — APPROVED** [review]

**Comment by AndrewShkrob** [comment]
@Jean-BaptisteC 
Please align your changes according to this PR: https://github.com/organicmaps/organicmaps/pull/5843.
I rechecked all strings, updated their names, and added proper comments to them.

File: data/strings/strings.txt (hunk 4 above)
Comment (AndrewShkrob) [reply data/strings/strings.txt:27047]: Everything here is correct. Just confusing comment. I updated it in https://github.com/organicmaps/organicmaps/pull/5843

**Comment by Jean-BaptisteC** [comment]
Done, I am waiting aa branch to be rebased, to only generate files for en and fr languages

**Comment by rtsisyk** [comment]
> Done, I am waiting aa branch to be rebased, to only generate files for en and fr languages

It has already been rebased.

**Comment by Jean-BaptisteC** [comment]
Can I have help to rebase correctly branch and fix conflicts?

**Comment by AndrewShkrob** [comment]
> Can I have help to rebase correctly branch and fix conflicts?

just remove your branch and create a new one from aa with the same name

```bash
git checkout fr_translations
git checkout -b fr_translations_tmp
git branch -D fr_translations
git checkout aa
git pull
git checkout -b fr_translations
git cherry-pick (insert here commit hash from fr_translations_tmp
git push --set-upstream origin fr_translations -f
```

**Comment by AndrewShkrob** [comment]
Also, you might need to delete your local `aa` branch and checkout it again because I squashed the commits, and it might not be possible to perform `git pull` now.

**Comment by Jean-BaptisteC** [comment]
Thanks for your help :)
Ready to merge

**Pull request merged**